src/services/edusharing/services/EduSharingConnectorV7.js

```diff
@@ -15,6 +16,7 @@ const ES_METADATASET =
 const ES_ENDPOINTS = {
 	NODE: `${Configuration.get('ES_DOMAIN')}/edu-sharing/rest/node/v1/nodes/-home-/`,
 	SEARCH: `${Configuration.get('ES_DOMAIN')}/edu-sharing/rest/search/v1/queries/-home-/${ES_METADATASET}/ngsearch`,
+	RENDERER: `${Configuration.get('ES_DOMAIN')}/edu-sharing/rest/rendering/v1/details/-home-/`,
 };
 
 class EduSharingConnector {
```

Comment:
Can you please make the 2 classes different over Naming? EduSharingConnectorV7 / EduSharingConnectorV6
It can be miss used or interpreted.

Reply:
The naming happened in exchange with Virgil. Furthermore, this is not part of this PR, because it comes from main-branch.

Reply:
If this part of the PR, or not is not really related to fix something. But if @virgilchiriac make the decission that it should keep the same name it is ok for me.